### ISSUE TITLE
Udacity Deep Learning Course

### ISSUE THREAD

**Issue by MOJTABAFA** [quinngroup/ToxoPlasma#13]
@naritapandhe 
As we discussed before , It was our first step to get familiar with deep learning in literature review step. 

I already finished it , please comment and let me know if you already finished this part.

**Comment by naritapandhe** [comment]
Yes, I have gone through the course.  I liked the CNN part of the course. They have provided an in-depth explanation for CNNs, but not for RNNs or LSTMs.
